b2sdk/stream/wrapper.py

```diff
@@ -0,0 +1,87 @@
+import io
+
+
+class BaseStreamWrapper(io.IOBase):
+    """
+    Base wrapper for a file-like object.
+
+    It does not implement public constructor so subclass have to define `stream` attribute
+    """
+
+    stream = NotImplemented
```

Comment:
https://docs.python.org/2.7/library/abc.html#abc.abstractproperty ?

Reply:
I knew that you would point it out - but I don't finished clean up yet - I just don't like `abc` overhead when I design intrefaces, so I quickly jest use NotImplemented and ususally I fix it on my private review before I push the code... I just didn't have time to finish this yet - so all comments about `abc` and docs are unnecessary distractions :)